molgenis/molgenis-emx2
# fix: input of newly selected refs

Fixes #3226 

Fixes the display of that label templates would only work on primary keys.
Works after selecting a new ref.
When editing already existing refs now use a backup display method in case the template does not resolve to a displayable result (see added unit test). This is because the full row data is not available at that stage. 

how to test:
- see #3226 or:
- go to the components lib 
- go to inputRefSelect (focus it if you want)
- see the 4th and 5th example


todo:
